### PR TITLE
FLEX-3319 Now saves device relay status after retrieval

### PR DESCRIPTION
**Pull request by Joep-Simons**

### CODE REVIEW

File: osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java [hunk 1]
```diff
@@ -276,4 +276,35 @@ public void setTransition(final String organisationIdentification, final String
...
+     * @param deviceIdentification
+     *            The device to update.
+     * @param deviceStatus
+     *            The device status to update the relay overview with.
+     */
+    private void updateDeviceRelayOverview(final Ssld device, final DeviceStatus deviceStatus) {
+        final List<RelayStatus> relayStatusses = device.getRelayStatusses();
+        for (final LightValue lightValue : deviceStatus.getLightValues()) {
+            boolean updated = false;
+            for (final RelayStatus relayStatus : relayStatusses) {
```
Comment (kevinsmeets) [inline osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java]: What if a device has no relayStatuses yet? Is it intended behaviour that  never a new relay status is created and saved, only exiting relay satus is updated?

File: osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java [hunk 2]
```diff
@@ -276,4 +276,35 @@ public void setTransition(final String organisationIdentification, final String
...
+     * Updates the relay overview from a device based on the given device
+     * status.
+     *
+     * @param deviceIdentification
+     *            The device to update.
+     * @param deviceStatus
+     *            The device status to update the relay overview with.
+     */
+    private void updateDeviceRelayOverview(final Ssld device, final DeviceStatus deviceStatus) {
+        final List<RelayStatus> relayStatusses = device.getRelayStatusses();
```
Comment (kevinsmeets) [inline osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java]: statuses is the correct spelling, not statusses

File: osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java (hunk 1 above)
Comment (Joep-Simons) [inline osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java]: Below code  verifies if the relaystatus was updated. If not, it's added.

Comment (Joep-Simons) [inline osgp-adapter-domain-publiclighting/src/main/java/com/alliander/osgp/adapter/domain/publiclighting/application/services/AdHocManagementService.java]: Apologies, I now see what you mean. I'm correcting this in the next commit.

**Comment by kevinsmeets** [comment]
I would like to reduce duplicate code, so I logged a new item FLEX-3407

**Pull request merged**